xml/System.Runtime.CompilerServices/Unsafe.xml

```diff
@@ -1581,12 +1957,22 @@ This check is conceptually similar to `(void*)(&left) < (void*)(&right)`.
         </Parameter>
       </Parameters>
       <Docs>
-        <typeparam name="T">The type of reference.</typeparam>
-        <param name="source">The reference to subtract the offset from.</param>
+        <typeparam name="T">The elemental type of the managed pointer.</typeparam>
+        <param name="source">The managed pointer to subtract the offset from.</param>
         <param name="byteOffset">The offset to subtract.</param>
-        <summary>Subtracts a byte offset from the given reference.</summary>
-        <returns>A new reference that reflects the subraction of byte offset from pointer.</returns>
-        <remarks>To be added.</remarks>
+        <summary>Subtracts a byte offset from the given managed pointer.</summary>
+        <returns>A new managed pointer that reflects the subtraction of byte offset from the source pointer.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+
+## Remarks
+
+The `byteOffset` parameter is the number of bytes to subtract from the `source` pointer. For example, given a source pointer _ptr_ of type `ref int`, the call `Unsafe.SubtractByteOffset<int>(ref ptr, (nuint)20)` will return a new pointer whose address points 20 bytes before _ptr_.
+
+If `elementOffset` is a calculated value rather than a hardcoded literal, callers should consider the possibility of integer overflow and should take appropriate precautions.
+
+]]></format>
+        </remarks>
       </Docs>
     </Member>
     <Member MemberName="Unbox&lt;T&gt;">
```

Comment:
not changes to `Unbox`?

Reply:
We recently updated the `Unbox` docs and I couldn't think of anything new to say. If you have suggestions I can definitely fold them in to this PR!